Controller/Controller.java

```diff
@@ -0,0 +1,469 @@
+            javafx.scene.image.Image icon= new javafx.scene.image.Image(ab);
+            stagep.getIcons().add(icon);
+
+            root12.setOnMousePressed(new EventHandler<MouseEvent>() {
+                @Override
+                public void handle(MouseEvent event) {
+                    x[0] = event.getSceneX();
+                    y[0] = event.getSceneY();
+                }
+            });
+
+            root12.setOnMouseDragged(new EventHandler<MouseEvent>() {
+                @Override
+                public void handle(MouseEvent event) {
+                    stagep.setX(event.getScreenX() - x[0]);
+                    stagep.setY(event.getScreenY() - y[0]);
+                }
+            });
+
+            root12.setOnMouseClicked(new EventHandler<MouseEvent>() {
+
+                @Override
+                public void handle(MouseEvent doubleClicked) {
+                    if(doubleClicked.getClickCount()==2){
+                        if(stagep.isFullScreen()){
+                            stagep.setFullScreen(false);
+                        }
+                        else
+                        {
+                            stagep.setFullScreen(true);
+                        }
+                    }
+                }
+            });
+            stagep.show();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    private void  FlowToGUI(String UI){
+        try {
+            String vi="/sample/view/"+UI+".fxml";
+            FXMLLoader loader=new FXMLLoader(getClass().getResource(vi));
+            Parent root=null;
+            Parent root5 = (Parent) loader.load();
+
+            String ads="/sample/images/Icon.png";
+            stage1=new Stage();
+            stage1.initModality(Modality.APPLICATION_MODAL);
+            if(UI =="aboutWindow"){
+                stage1.setTitle("ABOUT");
+                stage1.setResizable(false);
+            }
+            else if(UI == "playlist"){
+                stage1.setTitle("Playlist");
+            }
+            else if(UI == "helpWindow"){
+                stage1.setTitle("Help");
+                stage1.setResizable(false);
+            }
+            else if(UI == "album"){
+                stage1.setTitle("Album");
+            }
+            else if(UI == "artist"){
+                stage1.setTitle("Artist");
+            }
+            else if(UI == "myMusic"){
+                stage1.setTitle("Music");
+            }
+            else if(UI =="recentMedia"){
+                stage1.setTitle("RecentMedia");
+            }
+            else if(UI =="createPlaylist"){
+                stage1.setTitle("New Playlist");
+                stage1.initStyle(StageStyle.UNDECORATED);
+            }
+            else if(UI =="DatabaseInfo"){
+                stage1.setTitle("Database Info");
+                stage1.setResizable(false);
+            }
+            else if(UI =="favouriteTracks"){

```

Comment:
Our static analysis engine detects that your pull request includes a change from a Comparison of String objects using `==` or `!=`. According to [this description](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#es-comparison-of-string-objects-using-or-es-comparing-strings-with-eq), unless both strings are either constants in a source file, or have been interned using the `String.intern()` method, the same string value may be represented by two different String objects. Consider using the `equals(Object)` method instead.